content/en/includes/default-storage-class-prereqs.md

```diff
@@ -1,4 +1,4 @@
-You need to either have a dynamic PersistentVolume provisioner with a default
+You need to either have a [dynamic PersistentVolume provisioning](docs/concepts/storage/dynamic-provisioning/) with a default
```

Comment:
```suggestion
You need to either have a [dynamic PersistentVolume provisioning](/docs/concepts/storage/dynamic-provisioning/) with a default
```

Reply:
Hey thank you for reviewing I think provisioning is a better word here as here we are talking about the process and not the tool which does the process. Please correct me if I am wrong